components/layout_2020/positioned.rs

```diff
@@ -471,6 +508,10 @@ impl HoistedAbsolutelyPositionedBox {
                         containing_block,
```

Comment:
I wrote some tests but they are failing, because the logic stretches to this `containing_block`, but it should be the inset-modified containing block (we still need to actual containing block to compute the aspect ratio, though). This is an unrelated bug, `width: stretch` is already broken.

So I will try to address this in other PRs, let's wait for that and then recheck if there is some existing test that is passing.

Reply:
Understood! Thank you for clarifying

Reply:
So this should wait for #34430 , and then only use `Size::Stretch` if the alignment is stretch and neither inset property on that axis is `auto`.

Reply:
I will try to review #34430 today.